src/main/java/edu/KevinChen/advancedjava/assignment7/util/InvalidXMLException.java

```diff
@@ -0,0 +1,20 @@
+package edu.KevinChen.advancedjava.assignment7.util;
+
+/**
+ *  Used to signal invalid XML or other JAXB related issues.
+ */
+public class InvalidXMLException extends Exception{
+
+    /**
+     * Constructs a new exception with the specified detail message,
```

Comment:
this is good. 
what is the purpose of not including a constructor that takes no args?